projects/components/src/header/page/page-header.component.ts

```diff
@@ -159,3 +185,12 @@ export const enum PageHeaderContentAlignment {
   Column = 'column-alignment',
   Row = 'row-alignment'
 }
+
+/**
+ * @param Default - Default mode with no time range or refresh button
+ */
+export const enum PageHeaderDisplayMode {
+  Default = 'default',
+  WithTimeRange = 'with-time-range',
```

Comment:
Nit might rename these to clarify. Time range may or may not have refresh, and most commonly we show it so default feels off too. May make sense to just be verbose

Reply:
I'll revisit this part.